### DISCUSSION TITLE
Prop to limit characters in TextInput component

### DISCUSSION THREAD

**Discussion by mkf62** [mantinedev/mantine#7280] [Feature requests]
Similar to the [minlength and maxlength](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/maxlength) attributes on native HTML inputs, I was wondering if props could be added that would support such on the TextInput component? 

I am using the `use-field` hook to do this now, but it seems like a lot of extra code when something similar could be implemented through props as we see in the NumberInput, for instance. Just a thought.

**Comment by tylermatchett** [comment]
You can just use standard input attributes as they pass down through to the base input, so in this case you can use minlength or maxlength.
